doc/rbd/rbd-snapshot.rst

```diff
@@ -65,76 +71,97 @@ snapshots using the ``rbd`` command.
 Create Snapshot
 ---------------
 
-To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
-name and the image name.  ::
+To create a snapshot, use the ``rbd snap create`` command and specify the pool
+name, the snap name, and the image name:
```

Comment:
The suggested order was pool, image, snap -- it's the order in which they are specified on the command line, elsewhere on this page and also a natural progression from bigger to smaller.